tardis/simulation/base.py

```diff
@@ -431,13 +434,15 @@ def log_plasma_state(
         )
 
         logger.info("Plasma stratification:\n%s\n", plasma_state_log)
-        logger.info(f"t_inner {t_inner:.3f} -- next t_inner {next_t_inner:.3f}")
+        logger.info(
+            f"\n\tCurrent t_inner {t_inner:.3f}\n\tExpected t_inner for next iteration {next_t_inner:.3f}\n"
+        )
 
     def log_run_results(self, emitted_luminosity, absorbed_luminosity):
         logger.info(
-            f"Luminosity emitted = {emitted_luminosity:.5e} "
-            f"Luminosity absorbed = {absorbed_luminosity:.5e} "
-            f"Luminosity requested = {self.luminosity_requested:.5e}"
+            f"\n\tLuminosity emitted   = {emitted_luminosity:.5e}\n"
```

Comment:
and here it is 5 significant digits.

Reply:
Yes, I can change it to have 3 significant digits but this can be discussed 😉. For the time, I will change it to 3 digits as I didn't know if this needs to be changed or not 😅

Reply:
Changed to 3 decimal places for consistency 🚀